### PR TITLE
Billing docs updates for Pricing v2

### PR DESCRIPTION
**Pull request by danieltprice**
Preview:
https://neon-next-git-dprice-billing-updates-neondatabase.vercel.app/docs/introduction/about-billing

### CODE REVIEW

File: content/docs/introduction/about-billing.md [hunk 1]
```diff
@@ -1,46 +1,42 @@
...
-Find detailed definitions of the metrics that make up your bill and a cost breakdown for compute, storage, and data transfer rates for your region.
+Find out how billing works and the metrics that affect your usage allowances.
 
 <DetailIconCards>
 
-<a href="/docs/introduction/billing" description="Find detailed descriptions of the metrics that make up your bill" icon="trend-up">Billing metrics</a>
+<a href="/docs/introduction/billing-overview" description="Learn about plan allowances, extra usage, and monitoring usage" icon="chart-bar">How billing works</a>
 
-<a href="/docs/introduction/billing-rates" description="Reference table of billing rates per region" icon="hourglass">Billing rates</a>
+<a href="/docs/introduction/billing" description="Take a deep dive into the usage metrics behind plan allowances and extra usage" icon="trend-up">Usage metrics</a>
```
Comment (bgrenon) [inline content/docs/introduction/about-billing.md:28]: Great!

File: content/docs/introduction/billing-overview.md [hunk 2]
```diff
@@ -1,37 +1,80 @@
...
+Extra storage is only available with the [Scale](/docs/introduction/plans##scale) plan. Extra storage is billed for in units of 10 GiB. For example, the Scale plan has an allowance of 50 GiB that is included in the plan's monthly fee. If you go over 50 GiB of storage, you are automatically billed for extra storage in increments of 10 GiB for the price stated on our [pricing](https://neon.tech/pricing) page. For example, as soon as you go over your allowance, say by 1 GiB, you are billed for one 10 GiB unit of storage. If you go over by more than 10 GiB, you will be billed for two 10 GiB units of storage, and so on.
+
+### Compute
+
+Extra compute usage is available with the [Launch](/docs/introduction/plans##launch) and [Scale](/docs/introduction/plans##scale) plans. Extra compute usage is billed for by _compute hour_. For example, the Launch plan has an allowance of 1200 compute hours included in the plan's monthly fee. If you use additional compute hours, you are billed for those at the compute-hour price stated on our [pricing](https://neon.tech/pricing) page.
```
Comment (bgrenon) [inline content/docs/introduction/billing-overview.md:49]:
```suggestion
Extra compute usage is available with the [Launch](/docs/introduction/plans##launch) and [Scale](/docs/introduction/plans##scale) plans. Extra compute usage is billed by _compute hour_. For example, the Launch plan has an allowance of 1200 compute hours included in the plan's monthly fee. If you use additional compute hours, you are billed for those at the compute-hour price stated on our [pricing](https://neon.tech/pricing) page.
```

Comment (michrome) [reply content/docs/introduction/billing-overview.md:49]: Launch comes with 1,200 active hours (300 compute hours)

**Review by michrome — COMMENTED** [review]
Lovely! We just need to be crisp about hours. We bill in Compute hours. In that context our **docs** should frequently call out "active hours x endpoint size = compute hours".

Free = 5 compute hours for non-primary branches. Primary is available 24/7.
Launch = 300 compute hours.
Scale = 750 compute hours.

---
So I'm not contradicting my comments on Slack - for our **marketing** content I think active hours is more appropriate. This is because the Free plan is limited to 0.25 vCPU. Then it makes sense to keep that unit for plan comparison. Therefore for marketing:

Free = 20 active hours for non-primary branches. Primary is available 24/7.
Launch = _up to_ 1,200 active hours.
Scale = _up to_ 3,000 active hours.

File: content/docs/introduction/billing-sample.md [hunk 3]
```diff
@@ -36,8 +34,12 @@ Given the high number of connections used by this application, [connection pooli
...
 * **Storage size.** The amount of database storage currently used by your project. It includes the total volume of data across all branches plus the shared history. The database is now over 25 GiB and growing steadily with new written data as the user base grows.
-* **Written data.** This metric shows the volume of data actively written to database storage. The daily average of 0.15 GB results in a total of 4.4 GB for the month.
-* **Data transfer.** This refers to the amount of data transferred out of Neon. Daily average is 0.09 GB with a monthly total of 2.7 GB.
+
+### Which Pricing Plan fits best?
+
+With Neon pricing, two key metrics help you decide which [pricing plan](https://neon.tech/pricing) is right for you: compute time and storage.
+
+At roughly 718 compute hours for the month, this applicattion is well under the 1,200 hours/month limit for the Launch plan. However, with a storage size of 25 GiB, the storage needs for this application are well over the Launch plan limit of 10 GiB. That makes the Scale plan the right choice: 3,000 hours/month compute and 50 GiB storage.
```
Comment (bgrenon) [inline content/docs/introduction/billing-sample.md:42]:
```suggestion
At roughly 718 compute hours for the month, this application is well under the 1,200 hours/month limit for the Launch plan. However, with a storage size of 25 GiB, the storage needs for this application are well over the Launch plan limit of 10 GiB. That makes the Scale plan the right choice: 3,000 hours/month compute and 50 GiB storage.
```

File: content/docs/introduction/manage-billing.md [hunk 4]
```diff
@@ -1,54 +1,54 @@
...
+## Update your billing email
 
-If you are unable to add a payment method, please [open a support ticket](/docs/introduction/support).
+To update your billing email:
 
-### Missed payments
+1. Navigate to the Neon Console.
+1. Select your Profile.
+1. Select **Billing** from the menu.
+1. Select **Billing info**
```
Comment (bgrenon) [inline content/docs/introduction/manage-billing.md:48]:
```suggestion
1. Select **Billing info**.
```

File: content/docs/introduction/support.md [hunk 5]
```diff
@@ -4,34 +4,40 @@ enableTableOfContents: true
...
 
-If you're unable to access Neon's primary support channels (e.g., the support ticket form in the Neon Console or our [Discord Server](https://discord.com/invite/92vNTzKDGp)), you can use the following email as a fallback:
+## Enterprise support
 
-**Email**: `support@neon.tech`
+With Enterprise Support, you have everything offered with the Priority plan plus dedicated Customer Success Team support, and SLAs.
 
-Response times might be longer than our primary channels, so we appreciate your patience.
+<Admonition type="note">
+If you are a Launch, Scale, or Enterprise user and are unable to access the support ticket form in the Neon Console, you can use the following email address as a fallback:  `support@neon.tech`
```
Comment (bgrenon) [inline content/docs/introduction/support.md:42]:
```suggestion
If you are a Launch, Scale, or Enterprise user and are unable to access the support ticket form in the Neon Console, you can use the following email address as a fallback: `support@neon.tech`
```

File: content/docs/manage/backups.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ Neon does not yet provide support for configuring automated backups in the Neon
...
+Neon retains a history for all branches, allowing you to restore your data to a particular date and time or Log Sequence Number (LSN). The history retention period is configurable. The supported limits are 24 hours for [Neon Free Tier](/docs/introduction/free-tier) users, 7 days for Launch plan users, and 30 days for Scale plan users users. With this backup option, no action or automation is required. You can restore your data to a past state at any time by creating a database branch, which is a near-instant operation. This feature is referred to [Point-in-time restore](/docs/introduction/point-in-time-restore).
```
Comment (bgrenon) [inline content/docs/manage/backups.md:11]:
```suggestion
Neon retains a history for all branches, allowing you to restore your data to a particular date and time or Log Sequence Number (LSN). The history retention period is configurable. The supported limits are 24 hours for [Neon Free Tier](/docs/introduction/free-tier) users, 7 days for Launch plan users, and 30 days for Scale plan users. With this backup option, no action or automation is required. You can restore your data to a past state at any time by creating a database branch, which is a near-instant operation. This feature is referred to [Point-in-time restore](/docs/introduction/point-in-time-restore).
```

File: content/docs/manage/branches.md [hunk 7]
```diff
@@ -22,15 +22,15 @@ When working with branches, it is important to remove old and unused branches. B
...
 
 ## Non-primary branch
 
 Any branch not designated as the primary branch is considered a non-primary branch. You can rename or delete non-primary branches.
 
-- For [Neon Free Tier](/docs/introduction/free-tier) users, compute endpoints associated with non-primary branches are suspended if you exceed the Neon Free Tier  _compute active time_ limit of 100 hours per month.
-- For [Neon Pro Plan](/docs/introduction/pro-plan) users, default limits prevent more than 20 simultaneously active compute endpoints. Beyond that limit, a compute endpoint associated with a non-primary branch remains suspended.
+- For Neon Free Tier users, compute endpoints associated with non-primary branches are suspended if you exceed the Neon Free Tier  _compute active time_ limit of 100 hours per month.
+- For paying users users, default limits prevent more than 20 simultaneously active compute endpoints. Beyond that limit, a compute endpoint associated with a non-primary branch remains suspended.
```
Comment (bgrenon) [inline content/docs/manage/branches.md:33]:
```suggestion
- For paying users, default limits prevent more than 20 simultaneously active compute endpoints. Beyond that limit, a compute endpoint associated with a non-primary branch remains suspended.
```

**Review by bgrenon — APPROVED** [review]
Nicely done!

File: content/docs/connect/connection-pooling.md [hunk 8]
```diff
@@ -15,7 +15,7 @@ This topic describes Neon's default connection limits, how connection pooling wo
...
+In Neon, the size of your compute determines the `max_connections` setting. The formula used to calculate `max_connections` is `RAM in bytes / 9531392 bytes`. For a Neon Free Tier compute, which has 1 GB of RAM, this works out to approximately 100 connections. Larger computes offered with paid plans have more RAM and therefore support a larger number of connections. For example, a compute with 12 GB of RAM supports 1351 connections. You can check the `max_connections` limit for your compute by running the following query from the Neon SQL Editor or a client connected to Neon:
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/docs/connect/connection-pooling.md:18]: Covered in the content that follows in this section. This is a preamble to connection pooling & 10,000 concurrent connections support.

**Pull request merged**